packages/client/src/CollectionItem.ts

```diff
@@ -14,6 +14,17 @@ export class CollectionItem implements UploadableCollectionItem {
         this._locId = args.locId;
         this.locClient = args.locClient;
         this.clientItem = args.clientItem;
+
+        const logionClassifications = args.clientItem.termsAndConditions
+            .filter(element => element.type === "logion_classification")
+            .map(element => element as LogionClassification);
+        if(logionClassifications.length > 0) {
```

Comment:
Extra Logion Classifications are ignored - We should throw an error if more than one is found.

Reply:
Done